netdeploy/remote/deployedNetwork.go

```diff
@@ -104,12 +100,21 @@ func replaceTokens(original string, buildConfig BuildConfig) (expanded string, e
 	tokenPairs = append(tokenPairs, "{{CrontabSchedule}}", buildConfig.CrontabSchedule)
 	tokenPairs = append(tokenPairs, "{{EnableAlgoh}}", strconv.FormatBool(buildConfig.EnableAlgoh))
 	tokenPairs = append(tokenPairs, "{{DashboardEndpoint}}", buildConfig.DashboardEndpoint)
+	tokenPairs = append(tokenPairs, buildConfig.MiscStringString...)
 
 	expanded = strings.NewReplacer(tokenPairs...).Replace(original)
 
 	// To validate that there wasn't a typo in an intended token, look for obvious clues like "{{" or "}}"
-	if strings.Index(expanded, "{{") >= 0 || strings.Index(expanded, "}}") >= 0 {
-		return "", ErrDeployedNetworkTokenError
+	openIndex := strings.Index(expanded, "{{")
+	closeIndex := strings.Index(expanded, "}}")
+	if openIndex >= 0 || closeIndex >= 0 {
+		if openIndex < 0 {
+			openIndex = 0
+		}
+		if closeIndex < 0 {
+			closeIndex = len(expanded) - 2
+		}
+		return "", fmt.Errorf("config file contains unrecognized token: %s", expanded[openIndex:closeIndex+2])
```

Comment:
could you create a error type `type ErrDeployedNetworkTokenError struct` for that ? The consumer would be able to use `errors.As` to determine if it's an error of the given type.